src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.js

```diff
@@ -0,0 +1,172 @@
+import checkForSystemAdmin from '@salesforce/apex/PS_GatewayManagement.isSystemAdmin';
+
+export default class GePaymentGatewayManagement extends LightningElement {
+
+    showSpinner;
+    gatewayId;
+
+    isReadOnly = true;
+    errorMessage;
+
+    isElevateCustomer;
+    isSystemAdmin;
+    hasAccess;
+
+    CUSTOM_LABELS = { messageLoading, insufficientPermissions, commonAdminPermissionErrorMessage };
+
+    async connectedCallback() {
+        try {
+            this.isSystemAdmin = await checkForSystemAdmin();
+            this.isElevateCustomer = await checkForElevateCustomer();
+
+            this.hasAccess = !!(this.isElevateCustomer && this.isSystemAdmin);
+        } catch(ex) {
+            this.errorMessage = buildErrorMessage(ex);
+            this.isError = true;
+        }
+
+        if (this.hasAccess) {
+            this.gatewayId = await getGatewayIdFromConfig();
+        }
+    }
+
+    _isSuccess;
+    get isSuccess() {
+        return this._isSuccess;
+    }
+    set isSuccess(value) {
+        this._isSuccess = value;
+
+        if (value) { this.isError = false; }
+    }
+
+    _isError;
+    get isError() {
+        return this._isError;
+    }
+    set isError(value) {
+        this._isError = value;
+
+        if (value) { this.isSuccess = false; }
+    }
+
+    get noAccessErrorMessage() {
+        if (!this.isSystemAdmin) {
+            return this.CUSTOM_LABELS.commonAdminPermissionErrorMessage
+        }
+        // Temporary Hardcoded Text until this functionality is moved out of NPSP entirely
+        if (!this.isElevateCustomer) {
+            return 'You must be an Elevate customer to use this setting.';
+        }
+    }
+
+    handleEdit() {
+        this.isReadOnly = false;
+    }
+
+    handleCancel() {
+        this.template.querySelector("[data-id='gatewayIdField']").value = this.gatewayId;
+        this.resetAlert();
+        this.clearFieldErrors();
+
+        this.isReadOnly = true;
+    }
+
+    async handleSave(event) {
+        this.resetAlert();
+
+        let gatewayId = this.validateGatewayId();
+
+        if (isEmpty(gatewayId)) {
+            return;
+        }
+
+        this.showSpinner = true;
+
+        try {
+
+            await setGatewayId({ gatewayId: gatewayId});
+
+            this.isReadOnly = true;
+            this.isSuccess = true;
+            this.showSpinner = false;
+        } catch(ex) {
+            this.errorMessage = buildErrorMessage(ex);
+            this.showSpinner = false;
+            this.isError = true;
+        }
+    }
+
+    validateGatewayId() {
+        // Temporary Hardcoded Text until this functionality is moved out of NPSP entirely
+
+        let gatewayIdField = this.template.querySelector("[data-id='gatewayIdField']");
+
+        if (isEmpty(gatewayIdField.value)) {
+
+            gatewayIdField.setCustomValidity('Invalid Gateway ID.');
+            gatewayIdField.reportValidity();
+
+
+            this.errorMessage = 'Enter a valid gateway ID.'
```

Comment:
Should the error message be a custom label?

Reply:
@RadSFDeveloper We decided to not create custom labels as this work is temporary and will be replaced by Gateway Routing Rule sets.